package.json

```diff
@@ -248,7 +248,7 @@
         {
           "command": "extension.NAMESPACE(node-debug).toggleSkippingFile",
           "group": "navigation",
-          "when": "inDebugMode && debugType == 'node' && callStackItemType == 'stackFrame'"
+          "when": "inDebugMode && debugType == 'NAMESPACE(node-debug)' && callStackItemType == 'stackFrame'"
```

Comment:
This should be contributed for Chrome too (you can just remove the debugType)

Reply:
Done! Thanks